packages/shared-components/src/components/SearchBox/styles/SearchBoxSearchButton.js

```diff
@@ -39,13 +42,18 @@ export default styled.button`
     box-shadow: ${get('shadows.short')};
   }
 
+  &:disabled {
+    margin-left: -${ACTIVE_SIZE};
+  }
+
   &:disabled::before {
+    position: absolute;
     width: ${INACTIVE_SIZE};
     height: ${INACTIVE_SIZE};
     top: 0.25em;
     line-height: ${INACTIVE_SIZE};
     font-size: 1em;
-    left: calc(-${INACTIVE_SIZE} - ${get('spacing.small')});
+    left: 10px;
```

Comment:
It's confusing how this is 10px and the other one is 15px. Does the button end up moving when it changes from disabled to enabled or vice versa?

Reply:
the 10px makes the same space between the button (disabled) to the right border of the input. The 15px adjusts the position the big blue button has. 15px lines it right inside of the right border. The origin impl makes the blue button half outside of the input which looks strange to me.